src/config/options.rs

```diff
@@ -236,6 +236,9 @@ pub struct WidthHeuristics {
     // Maximum line length for single line if-else expressions. A value
     // of zero means always break if-else expressions.
     pub(crate) single_line_if_else_max_width: usize,
+    // Maximum line length for single line let-else statements. A value of zero means
+    // always format the divergent `else block` over multiple lines."
```

Comment:
leftover quotation mark?

```suggestion
    // always format the divergent `else block` over multiple lines.
```

As an aside, I think these should all be doc comments instead of normal comments.

Reply:
Thanks for catching that. It was late when I wrote this!